docs/code-reviews/process-guidance/author-guidance.md

```diff
@@ -22,6 +22,18 @@ Discuss design/code logic and address all comments as follows:
 - If you don't understand a comment, ask questions in the review itself as opposed to a private chat
 - If a thread gets bloated without a conclusion, have a meeting with the reviewer (call them or knock on door)
 
+## Use checklists
+
+At the moment of creating a pull request it is a very good idea to put in the description of it a checklist of expected objectives to achieve in the PR. This way reviewers can be able to focus on specific areas and also consider some other tasks as secondary according to this list.
+
+## Code should have annotations before the review
+
+If big files are something that can not be avoided then you need to include comments in order to make it easier to the teams to review, with clear comments developers can go faster in the process and also can identify goals for every code block.
```

Comment:
```suggestion
If you can't avoid large PRs, include explanations of the changes in order to make it easier for the reviewer to review the code, with clear comments the reviewer can identify the goal of every code block.
```

Reply:
Must say that I don't necessarily agree with this. The code should be able to speak for itself, through proper naming and through splitting large functions into smaller etc. If we need to add comments, the code should be refactored to be more readable. Comments will always get stale and incorrect over time as the code changes, but the devs forget about updating the comments.

We should however, add comments to the PR to describe the changes we have made, and why.

Reply:
I like the improvement, change made!